README.md

```diff
@@ -412,7 +412,7 @@ Services to securely store your Docker images.
 * [Atomic Registry](http://www.projectatomic.io/registry/) - Red Hat Atomic Registry is an open source enterprise registry based on the Origin and Cockpit projects, enhancing the Docker registry library.
 * [Azure Container Registry](https://azure.microsoft.com/de-de/services/container-registry/) Manage a Docker private registry as a first-class Azure resource
 * [CargoOS](https://github.com/RedCoolBeans/cargos-buildroot) - A bare essential OS for running the Docker Engine on bare metal or Cloud. By [@RedCoolBeans](https://github.com/RedCoolBeans)
-* [Cycle.io](https://cycle.io/) Bare-metal container hosting.
+* [Cycle.io](http://cycle.io/) Bare-metal container hosting.
```

Comment:
I tried this URL and it redirects automatically to https....

Reply:
... I'll drop this commit